File: app/src/main/java/com/example/photoaivideo/LibraryActivity.java
package com.example.photoaivideo;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class LibraryActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_library);

        Button btnImages = findViewById(R.id.btnImages);
        Button btnVideos = findViewById(R.id.btnVideos);

        btnImages.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(LibraryActivity.this, ImagesLibraryActivity.class));
            }
        });

        btnVideos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(LibraryActivity.this, VideosLibraryActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/example/photoaivideo/LibraryActivity.kt
package com.example.photoaivideo

import android.content.Intent
import android.os.Bundle
import android.widget.Button

class LibraryActivity : BasePermissionActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_library)
    }

    override fun onStoragePermissionGranted() {
        val btnImagesLibrary = findViewById<Button>(R.id.btnImagesLibrary)
        val btnVideosLibrary = findViewById<Button>(R.id.btnVideosLibrary)

        btnImagesLibrary.setOnClickListener {
            startActivity(Intent(this, ImagesLibraryActivity::class.java))
        }

        btnVideosLibrary.setOnClickListener {
            startActivity(Intent(this, VideosLibraryActivity::class.java))
        }
    }
}
